.github/CODEOWNERS

```
@@ -3,7 +3,7 @@
 # These owners will be the default owners for everything in the
 # repo. Unless a later match takes precedence, these owners will be
 # requested for review when someone opens a pull request.
-* @dav3r @dylanj1752 @felddy @hillaryj @INLGuy @itsmostafa @jsf9k @mcdonnnj @zenine07
+* @dav3r @dylanj1752 @felddy @hillaryj @INLGuy @itsmostafa @jsf9k @mcdonnnj @xlyk
```

@hillaryj is no longer with us (she's still alive but no longer works here):
```suggestion
* @dav3r @dylanj1752 @felddy @INLGuy @itsmostafa @jsf9k @mcdonnnj @xlyk
```

---

resolved: [307f993](https://github.com/cisagov/domain-manager-api/pull/516/commits/307f9933bfbe46e88adda997822c48c482556692)